challenge/development.txt

```diff
@@ -1,27 +0,0 @@
-Started out looking at the source of the top-level web page for jana.com.  Noted that there are triggers on scrolling to load up some content.  This implies some UI might be required to get at all the content.
```

Comment:
Yes.  Otherwise Selenium is just a huge performance hole for this task.  If you just 'curl' the top page, there is no content.